var/spack/repos/builtin/packages/py-adb-enhanced/package.py

```diff
@@ -17,15 +17,19 @@ class PyAdbEnhanced(PythonPackage):
 
     license("Apache-2.0")
 
-    version("2.5.10", sha256="9e913d09814ce99974c455a766c5b616a92bca551e657517d6e079882eb19bdb")
-    version("2.5.4", sha256="329ee2e0cfceaa41c591398b365d9acdfd45ffe913c64ac06e1538041986fffb")
-    version("2.5.3", sha256="5a1d5182d1a073b440e862e5481c7a21073eccc3cda7a4774a2aa311fee9bbdc")
-    version("2.5.2", sha256="055676156c1566b8d952b9fdfdd89fc09f2d5d1e3b90b4cdf40858ce9947e2ca")
```

Comment:
What happened to older versions? No wheels?

Reply:
They don't exist https://pypi.org/project/adb-enhanced/#history